ui/src/Components/feedback/Feedback.js

```diff
@@ -79,14 +80,15 @@ const Feedback = ({feedbackType, managerList}) => {
   const toggleFetch = () => {
     setFetch(!fetch);
     refetch({
-      variables: {employeeId: loggedInUser.employeeId},
+      variables: { employeeId: loggedInUser.employeeId },
     });
   };
 
   return (
     <div className="fb-main">
       <nav className="fb-navigation-bar">
         <div className={classes.root}>
+          <SearchBox></SearchBox>
```

Comment:
Add this to Thread.js  grid layout.  It makes more sense to have it inside Thread component.
<SearchBox></SearchBox> can be written as <SearchBox />

Reply:
Search Box will be present in the application even if there are no feedback messages. So will it not be better if we keep it under Feedback.js?